internal/flow/processor/basic.go

```diff
@@ -12,13 +12,19 @@ package processor
 
 import (
 	"context"
+	"time"
 
 	"trpc.group/trpc-go/trpc-agent-go/agent"
 	"trpc.group/trpc-go/trpc-agent-go/event"
 	"trpc.group/trpc-go/trpc-agent-go/log"
 	"trpc.group/trpc-go/trpc-agent-go/model"
 )
 
+const (
+	// Timeout for event completion signaling.
```

Comment:
exported: comment on exported const EventCompletionTimeout should be of the form "EventCompletionTimeout ..." (revive)